concepts/regular-expressions/about.md

```diff
@@ -0,0 +1,160 @@
+# About
+
+## 1. Regular Expressions in JavaScript
+
+A [Regular Expression][mdn-regular-expressions] (or Regex) is a sequence of characters that we can use to target and manipulate certain elements in strings. Hence, we can:
+
+- **Search** for a text in a string
+- **Replace** substrings in a string
+- **Extract** information from a string
+
+> 💡 JavaScript's regex flavor is part of the ECMA-262 standard for the language. This means that you don't have to worry about browser-specific compatibility.
+
+## 2. How to create Regular Expressions
+
```

Comment:
An introduction sentence about that there are 2 ways to create regex and when to use which would be nice. For performance concerns you can say "see below" but might be good to mention that the constructor can be used when building regex dynamically from strings.

Reply:
I am a bit confused here. This was marked resolved but there is no change visible.

Reply:
@junedev Oh I'm sorry if I forgot to change that! Please I truly appreciate every comment and suggestion you are writing, I tried to change one by one on my local files. The problem is that I am not really familiar with these suggestions, resolving and implementing workflow. 

Could you tell me what is the most standard procedure?

I completely agree with every comment 💪🏽If I didn't implement the suggestions is just because I didn't see it or I didn't put enough effort into it. Sorry about that. Next time I'll try to be more careful. 

As soon as I have time, I'll work on this introductory sentence and the mention for the constructor. 

All the love.